spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ControllerHelperServiceUtils.java

```diff
@@ -1,72 +1,19 @@
 package uk.ac.bbsrc.tgac.miso.spring;
 
-import static uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.isStringEmptyOrNull;
-
 import java.io.File;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
 
 import javax.servlet.http.HttpSession;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import net.sf.json.JSONArray;
-import net.sf.json.JSONObject;
-import net.sourceforge.fluxion.ajax.util.JSONUtils;
-
-import uk.ac.bbsrc.tgac.miso.core.data.Barcodable;
-import uk.ac.bbsrc.tgac.miso.core.data.Printer;
-import uk.ac.bbsrc.tgac.miso.service.PrinterService;
-
 /**
  * For when you need web utils, since this module can't access `miso-web`.
  *
  */
 public class ControllerHelperServiceUtils {
   private static final Logger log = LoggerFactory.getLogger(ControllerHelperServiceUtils.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "log" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)

Reply:
:+1: